plugins/permission-node/src/integration/createConditionFactory.ts

```diff
@@ -33,12 +36,17 @@ import { PermissionRule } from '../types';
  *
  * @public
  */
-export const createConditionFactory =
-  <TResourceType extends string, TParams extends any[]>(
-    rule: PermissionRule<unknown, unknown, TResourceType, TParams>,
-  ) =>
-  (...params: TParams): PermissionCondition<TResourceType, TParams> => ({
-    rule: rule.name,
-    resourceType: rule.resourceType,
-    params,
-  });
+export const createConditionFactory = <
+  TResourceType extends string,
+  TParams extends PermissionRuleParams = PermissionRuleParams,
+>(
+  rule: PermissionRule<unknown, unknown, TResourceType, TParams>,
+) => {
+  return (...args: [TParams]): PermissionCondition<TResourceType, TParams> => {
```

Comment:
```suggestion
  return (params: TParams): PermissionCondition<TResourceType, TParams> => {
```

Right? 😅

Reply:
Right 😅 . I think we made a later change that removed the need for what we were doing here. Have check this and it's all fine, there was a needed change to go with this a few lines down so will pop on another commit. 👍